rmf_fleet_adapter/src/full_control/main.cpp

```diff
@@ -467,11 +467,29 @@ struct Connections : public std::enable_shared_from_this<Connections>
           path_request_pub, mode_request_pub);
 
     const auto& l = state.location;
+    const auto& starts = rmf_traffic::agv::compute_plan_starts(
+        *graph, state.location.level_name, {l.x, l.y, l.yaw},
+        rmf_traffic_ros2::convert(adapter->node()->now()));
+    if (starts.empty())
+    {
+      RCLCPP_ERROR(
+        adapter->node()->get_logger(),
+        "Unable to compute a StartSet for robot [%s] using level_name [%s] and "
+        "location [%f,%f,%f] specified in its RobotState message. This can "
```

Comment:
Nitpick:

```suggestion
        "location [%f, %f, %f] specified in its RobotState message. This can "
```

Floating point numbers can be hard to visually recognize without any spacing between them.